package.json

```diff
@@ -44,9 +44,9 @@
     "jsii-diff": "^1.18.0",
     "jsii-docgen": "^1.3.2",
     "jsii-pacmak": "^1.18.0",
-    "jsii-release": "^0.2.3",
+    "jsii-release": "^0.2.11",
```

Comment:
Upgrade to `0.2.12` to get https://github.com/aws/jsii-release/pull/25

Reply:
Done